sdk/translation/azure-ai-translation-text/setup.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+from io import open
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "azure-ai-translation-text"
+version = "1.0.0b1"
```

Comment:
we don't usually hardcode the version, we get it from the _version.py. See the document translation setup.py for how to get it

Reply:
Done